zigzag/cli.py

```diff
@@ -19,23 +19,15 @@
               is_flag=True,
               default=False,
               help='Pretty print XML on schema violations to stdout')
-@click.option('--qtest-test-cycle', '-t',
-              type=click.STRING,
-              default=None,
-              help='Specify a test cycle to use as a parent for results.')
-@click.option('--test-runner', '-tr',
-              type=click.Choice(['pytest-zigzag', 'tempest']),
-              default='pytest-zigzag',
-              help='Specify the tool that generated the xml to be processed')
+@click.argument('zigzag_config_file', type=click.Path(exists=True))
 @click.argument('junit_input_file', type=click.Path(exists=True))
-@click.argument('qtest_project_id', type=click.INT)
-def main(junit_input_file, qtest_project_id, qtest_test_cycle, pprint_on_fail, test_runner):
+def main(junit_input_file, zigzag_config_file, pprint_on_fail):
```

Comment:
nit: can you order these argument in the same order that they occur on the command line?  

Example:
`def main(zigzag_config_file, junit_input_file, pprint_on_fail):`